dev/com.ibm.ws.concurrent_fat/fat/src/com/ibm/ws/concurrent/fat/FATSuite.java

```diff
@@ -15,5 +15,8 @@
 import org.junit.runners.Suite.SuiteClasses;
 
 @RunWith(Suite.class)
-@SuiteClasses({ EEConcurrencyUtilsFATTest.class })
+@SuiteClasses({
+                EEConcurrencySpecTest.class,
```

Comment:
EEConcurrencySpecTest is the most commonly used test bucket for concurrency.  It is lightweight, extremely fast-running, and has excellent coverage.  It shouldn't be combined with a slower, more specialized bucket for server config updates.